app/paste.c

```diff
@@ -15,12 +15,24 @@
 #include "zsv_command.h"
 
 static int zsv_paste_usage() {
-  static const char *usage = "Usage: paste <filename> [<filename> ...]\n"
-                             "\n"
-                             "Options:\n"
-                             "  -h,--help            : show usage\n";
-  printf("%s\n", usage);
-  return 1;
+  /* clang-format off */
+  static const char *usage[] = {
+    "Usage: paste <filename> [<filename> ...]",
+    "",
+    "Horizontally paste two tables together: given inputs X, Y, ... of N rows",
+    "outputs 1...N rows, where each row i contains:",
+    "  row i of input X,",
+    "  followed by row i of input Y,",
+    "  [followed by ...]",
+    "",
+    "Options:",
+    "  -h,--help : show usage",
+    NULL
```

Comment:
**FYI**
Adding a trailing comma at the end of last element will make `clang-format` to preserve this format.
No need to turn the formatting off here.

Reply:
Good to know, will do that thank you!